src/playlist.tsx

```diff
@@ -49,6 +52,7 @@ export class Playlist extends KalturaPlayer.core.BasePlugin {
       } else if (this._activePresetName === '') {
         this._activePresetName = state.shell.activePresetName;
       }
+      this._addSidePanel();
```

Comment:
why do you need this here as well if it will be called in loadMedia ?

Reply:
because in mobile can change portrait and landscape and i wanted the ui to change accordingly